medium/io-js-week-of-february-13th-2015.md

```diff
@@ -61,29 +61,29 @@ View the [original article](https://medium.com/@mikeal/how-io-js-built-a-146-per
 * [`iojs-tw`](https://github.com/iojs/iojs-tw) Taiwan Community
 * [`iojs-uk`](https://github.com/iojs/iojs-uk) Ukranian Community
 
-## io.js and Node.js
-View the [original article](https://medium.com/@iojs/io-js-and-a-node-js-foundation-4e14699fb7be) on Medium.
-* Scott Hammond, CEO of Joyent, expressed his desire to bring io.js back to the node.js.
+## io.js e Node.js
+Leggi [l'articolo originale](https://medium.com/@iojs/io-js-and-a-node-js-foundation-4e14699fb7be) su Medium.
+* Scott Hammond, CEO di Joyent, esprime il desiderio di riportare io.js in node.js.
 
-#### In only a few months io.js... 
-* Has grown to 23 active core team members
-* Has several working groups
-* Has 29 language localization teams,
-* Has drawn more contributors to the project than we’ve ever had in the history of node.js, and
-* Has been able to release quality software at a good pace with the support of an exceptional community.
+#### In pochi mesi di io.js... 
+* Il team _core_ è cresciuto fino a 23 collaboratori attivi,
+* si sono creati molti gruppi di lavoro,
+* è arrivato ad avere 29 team di localizzazione,
+* ha portato più collaboratori al progetto di quanti ce ne siano stati nella storia di node.js, e
+* siamo riusciti a rilasciare software di qualità ad un buon passo grazie al supporto di una community eccezionale.
```

Comment:
Come "camminare di buon passo" ma effettivamente non rende come in inglese, vedo di inventarmi qualcosa che renda meglio l'idea